stardis/io/model/MARCS.py

```diff
@@ -0,0 +1,138 @@
+import pandas as pd
+import gzip
+import re
+from dataclasses import dataclass
+
+
+@dataclass
+class MARCSModel(object):
+    """
+    Class to hold a MARCS model. Holds a dict of the meta information and a pandas dataframe of the contents.
+
+    """
+
+    meta: dict
+    data: pd.DataFrame
+
+
```

Comment:
make a constant that describes the data format

Reply:
Code rewritten as described.